### PR TITLE
adding a new function

### PR DESCRIPTION
**Pull request by jinshan1979**
1. update the original function name
2. adding a new function with old name

### CODE REVIEW

File: csharp/dotnet-windows/console/Samples/Samples/Program.cs [hunk 1]
```diff
@@ -67,7 +72,7 @@ static void Main(string[] args)
 
         // Pronunciation assessment configured with json
         // See more information at https://aka.ms/csspeech/pa
-        public static async Task GetProsodyResult()
+        public static async Task GetAssessmentResultWithScenarioID()
```
Comment (wangkenpu) [inline csharp/dotnet-windows/console/Samples/Samples/Program.cs]: `GetAssessmentResultsWithScenarioID` -> `GetAssessmentResultsWithScenarioId`

File: csharp/dotnet-windows/console/Samples/Samples/Program.cs [hunk 2]
```diff
@@ -209,5 +214,70 @@ public static async Task GetContentResult()
...
+        // Pronunciation assessment with enableProsodyAssessment
+        // See more information at https://aka.ms/csspeech/pa
+        public static async Task GetProsodyResult()
+        {
+            // Creates an instance of a speech config with specified subscription key and service region.
+            // Replace with your own subscription key and service region (e.g., "westus").
+            var config = SpeechConfig.FromSubscription("YourSubscriptionKey", "YourServiceRegion");
+            
+            //set to true to get the prosody value
+            bool enableProsodyAssessment = true;
```
Comment (wangkenpu) [inline csharp/dotnet-windows/console/Samples/Samples/Program.cs]: For `GetProsodyResult`, `enableProsodyAssessment` should always be `true`. So you can use hard code in line 241.

```C#
 string jsonConfig = "{\"GradingSystem\":\"HundredMark\",\"Granularity\":\"Phoneme\",\"EnableMiscue\":\"true\", \"EnableProsodyAssessment\":\"true\"}";
```

File: csharp/dotnet-windows/console/Samples/Samples/Program.cs (hunk 1 above)
Comment (jinshan1979) [reply csharp/dotnet-windows/console/Samples/Samples/Program.cs]: fixed

File: csharp/dotnet-windows/console/Samples/Samples/Program.cs (hunk 2 above)
Comment (jinshan1979) [reply csharp/dotnet-windows/console/Samples/Samples/Program.cs]: fixed and removed the variable

File: csharp/dotnet-windows/console/Samples/Samples/Program.cs [hunk 3]
```diff
@@ -30,7 +30,7 @@ static void Main(string[] args)
                 Console.WriteLine(choose);
                 Console.WriteLine("");
                 Console.WriteLine("1. Pronunciation Assessment with Content.");
-                Console.WriteLine("2. Pronunciation Assessment with Scenario ID.");
+                Console.WriteLine("2. Pronunciation Assessment with Scenario Id.");
```
Comment (wangkenpu) [inline csharp/dotnet-windows/console/Samples/Samples/Program.cs]: Here "ID" is in a sentence and is not a variable or function name. So please do not change "ID" to "Id".

File: csharp/dotnet-windows/console/Samples/Samples/Program.cs [hunk 4]
```diff
@@ -238,7 +235,7 @@ public static async Task GetProsodyResult()
             var audioConfig = AudioConfig.FromWavFileInput(wavePath);
 
             // create pronunciation assessment config, set grading system, granularity and if enable miscue based on your requirement.
-            string jsonConfig = "{\"GradingSystem\":\"HundredMark\",\"Granularity\":\"Phoneme\",\"EnableMiscue\":true, \"enableProsodyAssessment\":\"" + enableProsodyAssessment + "\"}";
+            string jsonConfig = "{\"GradingSystem\":\"HundredMark\",\"Granularity\":\"Phoneme\",\"EnableMiscue\":true, \"enableProsodyAssessment\":\"true\"}";
```
Comment (wangkenpu) [inline csharp/dotnet-windows/console/Samples/Samples/Program.cs]: For other configure, the first letter is capitalized. So please change `enableProsodyAssessment` to `EnableProsodyAssessment`.

File: csharp/dotnet-windows/console/Samples/Samples/Program.cs (hunk 3 above)
Comment (jinshan1979) [reply csharp/dotnet-windows/console/Samples/Samples/Program.cs]: udpated

File: csharp/dotnet-windows/console/Samples/Samples/Program.cs (hunk 4 above)
Comment (jinshan1979) [reply csharp/dotnet-windows/console/Samples/Samples/Program.cs]: udpated

**Review by wangkenpu — APPROVED** [review]

**Pull request merged**